azure-pipelines.yml

```diff
@@ -3,17 +3,25 @@ jobs:
   pool:
     vmImage: xcode9-macos10.13
   steps:
+    - bash: env | grep ^CI_
```

Comment:
This is failing in CI. Maybe there aren't any vars that begin with CI at this point?